mobius-rx3/src/main/java/com/spotify/mobius/rx3/Transformers.java

```diff
@@ -0,0 +1,201 @@
+   * the requested effect object.
+   *
+   * @param doEffect the {@link Consumer} to be run every time the effect is requested
+   * @param scheduler the {@link Scheduler} to be used when invoking the consumer
+   * @param <F> the type of Effect this transformer handles
+   * @param <E> these transformers are for effects that do not result in any events; however, they
+   *     still need to share the same Event type
+   * @return an {@link ObservableTransformer} that can be used with a {@link
+   *     RxMobius.SubtypeEffectHandlerBuilder}.
+   */
+  static <F, E> ObservableTransformer<F, E> fromConsumer(
+      final Consumer<F> doEffect, @Nullable final Scheduler scheduler) {
+    return new ObservableTransformer<F, E>() {
+      @Override
+      public ObservableSource<E> apply(Observable<F> effectStream) {
+        return effectStream
+            .flatMapCompletable(
+                new Function<F, CompletableSource>() {
+                  @Override
+                  public CompletableSource apply(final F effect) {
+                    Completable completable =
+                        Completable.fromAction(
+                            new Action() {
+                              @Override
+                              public void run() throws Throwable {
+                                doEffect.accept(effect);
+                              }
+                            });
+                    return scheduler == null ? completable : completable.subscribeOn(scheduler);
+                  }
+                })
+            .toObservable();
+      }
+    };
+  }
+
+  /**
+   * Creates an {@link ObservableTransformer} that will flatten the provided {@link Function} into
+   * the stream as an {@link Observable} every time it receives an effect from the upstream effects
+   * observable. This will result in calling the function on the specified scheduler, and passing it
+   * the requested effect object then emitting its returned value.
+   *
+   * @param function the {@link Function} to be invoked every time the effect is requested
+   * @param scheduler the {@link Scheduler} to be used when invoking the function
+   * @param <F> the type of Effect this transformer handles
+   * @param <E> the type of Event this transformer emits
+   * @return an {@link ObservableTransformer} that can be used with a {@link
+   *     RxMobius.SubtypeEffectHandlerBuilder}.
+   */
+  static <F, E> ObservableTransformer<F, E> fromFunction(
+      final Function<F, E> function, @Nullable final Scheduler scheduler) {
+    return new ObservableTransformer<F, E>() {
+      @Override
+      public ObservableSource<E> apply(Observable<F> effectStream) {
+        return effectStream.flatMap(
+            new Function<F, ObservableSource<E>>() {
+              @Override
+              public ObservableSource<E> apply(@NonNull F f) {
+                Observable<E> eventObservable =
+                    Observable.fromSupplier(
```

Comment:
In rxmobius2 and rxmobius we are using `fromCallable` and then calling `function.apply` in rx1 the `Function` signature did not throw any exception, in rx2 the `Function` it throws an `Exception` and in rx3 it throws a `Throwable` but the method `call` of `Callable` expects an `Exception` so there is a type mismatch. So I opted out to use a supplier here. 
<img width="1302" alt="Screen Shot 2020-04-22 at 7 53 14 AM" src="https://user-images.githubusercontent.com/802308/79978859-989e4680-846e-11ea-8e33-8548d5eac2b5.png">
<img width="924" alt="Screen Shot 2020-04-22 at 7 53 22 AM" src="https://user-images.githubusercontent.com/802308/79978860-989e4680-846e-11ea-9a1d-69bab3b9c159.png">